index.html

```diff
@@ -20,377 +20,273 @@
-           <span>Html</span>
+  
+        <div class="card-2">
+          <div class="fourth-card">
+            <div class="fourth-block">
+              <h1>Profesional Art Printing Data More</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-4">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-          <div class="four-tag2">
-           <span>bootstrap</span>
+          <div class="fifth-card">
+            <div class="fifth-block">
+              <h1>Profesional Art Printing Data More</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-5">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-          <div class="four-tag3">
-           <span>Ruby</span>
+          <div class="sixth-card">
+            <div class="sixth-block">
+              <h1>Website Portfolio</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-6">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
         </div>
-
-
-
-
-        
-
-      </section>
-
-      
+      </div>
 
     </section>
-    <section class="about">
-      <h1>
-        About Me
-      </h1>
 
-      
-      <span>
-          Hello I’m a software developer! I can help you build a product , 
+    <section class="about">
+      <div class="about-me">
+        <h1>About me</h1>
+        <p>Hello I’m a software developer! I can help you build a product , 
           feature or website Look through some of my work and experience! 
           If you like what you see and have a project you need coded, 
           don’t hestiate to contact me.
-      </span>
+        </p>
 
-      <div class="about-resume">
-        <small>
-          Get My Resume
-        </small>
+        <div class="about-btn">
+          <h4>Get My Resume</h4>
+        </div>
       </div>
-
-      <div class="about-image">
-        <img src="assets/about-design.png" alt=""/>
+      <div class="about-img">
+        <img src="assets/Frame.png" alt=""/>
```

Comment:
- [ ] As per the project requirement you should stick to the design as much as possible. Please decrease the height of the line. See the image below for reference.

### Your design
![your](https://user-images.githubusercontent.com/81727988/206646325-54fefaf4-2b10-4cf9-8b7a-03bcd90d075d.png)
### Figma design
![gima](https://user-images.githubusercontent.com/81727988/206646362-4c568713-0c32-4616-8714-1a53c106242d.png)

Reply:
- [ ] I just realised that your mobile version is not really completed and all your cards are not displayed , kindly fix this for your project to match the requirement for this project 
![Screenshot from 2022-12-09 23-49-48](https://user-images.githubusercontent.com/88894525/206801537-3dc90710-a539-4c53-bb0a-e1fb18e98f0a.png)